gui/gamesetup/gamesetup~!extra_commands.js

```diff
@@ -348,16 +349,58 @@ g_NetworkCommands['/countdown'] = input =>
 
 g_NetworkCommands['/gameName'] = text =>
 {
-	if (!g_IsController || !Engine.HasNetServer())
-		return;
-	if (!g_SetupWindow.controls.lobbyGameRegistrationController)
-		return
-	text = `${text}`
-	g_SetupWindow.controls.lobbyGameRegistrationController.serverName = text
-	selfMessage(`Game name changed to: ${text}`)
-	g_SetupWindow.controls.lobbyGameRegistrationController.sendImmediately()
+	return setGameNameInLobby(text);
+}
+
+
+g_NetworkCommands["/jitsi"] = text =>
+{
+	// selfMessage(`meet.jit.si/anyNameYoutWantHere`);
+	selfMessage(
+	  `Jitsi Meet is a fully encrypted, 100% open source video conferencing solution that you can use all day, every day, for free — with no account needed.`
+	)
+
+
+	let gitHubLinkAutoCivModificationSL5 = 'https://github.com/sl5net/autociv';
+
+	let textBest = `Jitsi is best for having a quick team call without setup process. could used as audio chat for you 0ad-team. `
+	selfMessage(
+		textBest
+	)
+
+	selfMessage(
+	  `write /link to open a link in your WebBrowser if you have autoCiv-mod installed to open a link more easy.`
+	)
```

Comment:
Remove all this, not necessary for the user

Reply:
i need to explain very often to user what it is. so i don't want remove it.